### PR TITLE
Bootcamp first layout

### PR DESCRIPTION
**Pull request by borislavstoychev**

### CODE REVIEW

File: schema.prisma [hunk 1]
```diff
@@ -66,6 +66,20 @@ model Person {
...
+  status  String
+  title  String
+  email   String
+  message String
+  date  DateTime
+  firstName String?
+  lastName String?
+  @@map("bootcamp")
+}
+
```
Comment (kachar) [inline schema.prisma:82]: After modifying the `schema.prisma` file you need to follow the procedure of generating a migration and rebuilding the schema:

```
yarn prisma migrate dev

yarn generate-schema
```

With these steps your PR will be complete

**Review by kachar — CHANGES_REQUESTED** [review]
Well done @borislavstoychev 

This PR brings almost all the required pieces to be an actual contribution to the project

Please check the comment about the auto-generated SQL migrations as they are important part of the process

File: schema.prisma (hunk 1 above)
Comment (borislavstoychev) [reply schema.prisma:82]: Thanks a lot!

**Review by kachar — APPROVED** [review]
@borislavstoychev Well done, this PR contains all the necessary features of a production code!

Thanks for investing your time in that! I think we're ready to move on to the next real challenges!

**Comment by kachar** [comment]
Closing the PR as it's demo code

**Pull request closed without merging**